src/cloudant/database.py

```diff
@@ -196,52 +187,54 @@ def get_design_document(self, ddoc_id):
 
     def get_view_result(self, ddoc_id, view_name, **kwargs):
         """
-        _get_view_result_
-
-        Returns a Result object based on the design document
+        Retrieves a Result object based on the design document
         and view name.  If you intend to iterate through the
-        result, do not use skip and/or limit in the kwargs as
+        result, do not use ``skip`` and/or ``limit`` in the kwargs as
         that is handled in the Result.  If you would like to
         manage paging and iteration manually over the result
-        then try the get_view_raw_result method instead.
+        then use the
+        :func:`~cloudant.database.CouchDatabase.get_view_raw_result`
+        method instead.
 
-        For example to retrieve the default Result object based
-        on a design document view do:
+        For example to retrieve the default Result object based on a
+        design document view do:
 
-        db.get_view_result('_design/ddoc_id_001', 'view_001')
+        .. code-block:: python
+
+            db.get_view_result('_design/ddoc_id_001', 'view_001')
 
         But to retrieve a customized Result object based on the
         same design document view do something like:
 
-        db.get_view_result('_design/ddoc_id_001', 'view_001',
-            include_docs=True, reduce=False)
-
-        For more detail on slicing and iteration using a Result
-        object, refer to the Result Class docstring.
-
-        :param ddoc_id: Design document id
-        :param view_name: Name of the view
-        :param **kwargs: Parameters to index the query results by
-            Valid parameters include:
-
-            descending bool
-            endkey string or array
-            endkey_docid  string
-            group bool
-            group_level ??
-            include_docs bool
-            inclusive_end  bool
-            key string
-            keys list
-            reduce  boolean
-            stale   enum(ok, update_after)
-            startkey  string or array
-            startkey_docid  string
+        .. code-block:: python
+
+            db.get_view_result('_design/ddoc_id_001', 'view_001',
+                include_docs=True, reduce=False)
+
+        For more detail on slicing and iteration, refer to the
+        :class:`~cloudant.result.Result` documentation.
+
+        :param str ddoc_id: Design document id
+        :param str view_name: Name of the view
+        :param bool descending: Valid parameter to index query results by
```

Comment:
I find "Valid parameter to index query results by" a bit confusing.
Firstly, why do we have to say "valid" surely all the documented parameters are valid?
Secondly, is it commonly called indexing the result?
I feel like many of these parameters are for sorting query results (e.g. `descending` `group`) whereas others are filters (`keys`) and others are directives (`reduce` `stale`).